src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      onsubmit="onSubmit()"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="part-form personal-info">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname-input"
+            id="surname"
+        >
+        </div>
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+           type="text"
+           name="name-input"
+           id="name">
+        </div>
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            value="12"
+            type="number"
+            name="age-input"
```

Comment:
use camelCase for name attribute. Or just don't use word 'input'
```suggestion
            name="age"
```